manifests/profile/www_lib/vhosts/default.pp

```diff
@@ -0,0 +1,46 @@
+
+# Copyright (c) 2019 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# nebula::profile::www_lib::vhosts::default
+#
+# Default (redirection) vhosts for www.lib.umich.edu environment
+#
+# @example
+#   include nebula::profile::www_lib::vhosts::default
+
+class nebula::profile::www_lib::vhosts::default (
+  String $prefix,
+  String $domain,
+  String $vhost_root,
```

Comment:
should get rid of this parameter for `www_lib::vhost`; it's only used to construct the default docroot and to allow access to `/www/www.lib/cgi`. It also looks like the ScriptAlias for /cgi is missing, and it seems unlikely that ScriptAlias is really needed by default for all vhosts (rather than just www.lib.umich.edu)